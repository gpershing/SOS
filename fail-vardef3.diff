diff: tests/fail-vardef3.err: No such file or directory
