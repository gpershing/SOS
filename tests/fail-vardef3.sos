x: float = 2.0
b: bool = x
