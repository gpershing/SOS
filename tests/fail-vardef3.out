Could not resolve type when defining b (Found float, expected bool)
